### DISCUSSION TITLE
A couple of questions before I install the` Fossify Gallery` app from F-Droid Repo

### DISCUSSION THREAD

**Discussion by rajibando** [FossifyOrg/General-Discussion#384] [Q&A]
Dear Team Fossify Gallery,

May please this post be perused: [Is there an image perspective correction tool in F-Droid Repo?](https://forum.f-droid.org/t/is-there-an-image-perspective-correction-tool-in-f-droid-repo/29530)

An unrestrained Crop tool and a perspective correction tool would help a great deal. A separate, tiny image-editing tool would greatly help.

**Comment by Aga-C** [comment]
We don't have it planned, but if you like, you can raise a feature request.

**Reply by rajibando** [reply]
Thank you very much for your prompt reply.

Do you have a special section for your users to post a Feature Request?

I am a Debian user. I have been using GIMP for the last 14 years. It has all the required tools. But no developer seems interested to have a very truncated Image Editor from the source and port that editor to F-Droid ecosystem as a separate, tiny app.

**Reply by Aga-C** [reply]
Issues → New issue → Feature request

**Reply by rajibando** [reply]
> Issues → New issue → Feature request
Ok. Thanks, Mr. Agni C.
Posted as a Feature Request,  #371.
Best wishes.

**Reply by Aga-C** [reply]
Fossify Gallery already has an editor, and there are already many feature requests about adding new functions to it. Your feature request is too vague, especially that we ask to describe only one feature in a request.

Please modify your feature request to be only about cropping with perspective correction.

**Reply by rajibando** [reply]
I had mentioned the request on the subject line itself. But ok. I have put the said lines within the body also.
Thank you for pointing out the need to polish the feature request.
